sources/packages/api/src-sql/User/Add-col-user-isactive.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "users" ADD COLUMN IF NOT EXISTS "is_active" BOOLEAN NULL DEFAULT TRUE;
```

Comment:
I believe that "is_active" should not be nullable.

Reply:
that I made the default value as TRUE, If that doesn't make i will update the code